app/model/Exceptions.scala

```diff
@@ -72,7 +72,7 @@ object Exceptions {
 
   case class IncorrectStatusInApplicationException(reason: String) extends Exception(reason)
 
-  case class InvalidStatusException(reason: String) extends Exception(reason)
+  case class InvalidStatusAndProgressStatusException(reason: String) extends Exception(reason)
```

Comment:
What I am trying to say with that exception is review your applicationStatus parameter and you progressStatus param because you pass a wrong combination.